docs/userguide.rst

```diff
@@ -384,8 +384,10 @@ follows:
 
 .. code-block:: console
 
+    $ # create new virtual environment
+    $ virtualenv myreana
+    $ source myreana/bin/activate
     $ # install REANA client:
-    $ mkvirtualenv reana-client
```

Comment:
You may also want to remove the `workon` command that is used elsewhere in the script